cmd/batch.go

```diff
@@ -318,21 +343,33 @@ func (b *standardBatchPipeline) GetChanges(
 func (b *standardBatchPipeline) WriteTemplate(
 	writer io.Writer,
 	template string,
-	pregap bool,
+	pregapLines int64,
+	postgapLines int64,
 	templateData interface{},
 ) error {
 	if template == "" {
 		return nil
 	}
 
-	if pregap {
-		_, err := writer.Write([]byte("\n"))
+	if pregapLines > 0 {
```

Comment:
After looking at the test I remember we use this newline write to add newlines between elements like changes and kinds. What we really want this to do is add additional empty lines of white space, so we will need a solution for that. I have an idea but if you have any opinions I would like to know since mine feels less than ideal.

Reply:
I noticed as well about this, my solution for this was the default tag for the elements we expect a new line

Reply:
My issue with that is the disconnect that the value 1 should mean one empty line, not just using the next line.
eg: `config.Newlines.AfterChange: 1` should do:
```markdown
- my change

- my second change
```

So I think the goal would be that the value of 0 for all newlines would produce an output with no empty lines anywhere. But that there would still be newlines inserted between elements that needed them, but no extras. For completeness I would say making sure the file ends with a newline as files should.

Reply:
The plus one was my idea, an extra bool seems weird I agree. I think if we just add one to all "Before" values except for version we should be good.
We need to add one parameter for the end of file in case you might want an extra line or two for post processing.
Additionally, I think we need to add new line options for the `merge` command as well.